test/molecule-role/molecule/default/tests/test_receiver.py

```diff
@@ -13,3 +15,35 @@ def test_etc_docker_directory(host):
 def test_docker_compose_file(host):
     f = host.file('/home/ubuntu/docker-compose.yml')
     assert f.is_file
+
```

Comment:
should we check if the 3 containers we expect are running ?
and/or check the receiver/health to be OK ?

Reply:
We could,  i would say molecule is in charge of the machines,  so i'd not second guess that. 
Receiver health is also not that important to me, we test the integration, not the operating of the receiver